shipment_advice/views/res_config_settings.xml

```diff
@@ -8,59 +8,55 @@
         <field name="model">res.config.settings</field>
         <field name="inherit_id" ref="stock.res_config_settings_view_form" />
         <field name="arch" type="xml">
-            <xpath
-                expr="//field[@name='group_stock_adv_location']/ancestor::div[hasclass('o_setting_box')]"
-                position="after"
-            >
-                <div class="col-12 col-lg-6 o_setting_box">
-                    <div class="o_setting_left_pane" />
+            <xpath expr="//setting[@id='use_own_routes']" position="after">
+                <setting
+                    id="backorder_policy"
```

Comment:
As we are on a generic form view (config panels) better to use an ID that won't conflict with other modules installed: `shipment_advice_backorder_policy`